packages/sanity/src/core/studio/copyPaste/transferValue.ts

```diff
@@ -633,13 +633,15 @@ async function collateObjectValue({
         if (options.client && targetSchemaType.options?.filter) {
           const getClient = (clientOptions: {apiVersion: string}) =>
             (options.client as SanityClient).withConfig(clientOptions)
-          const isMatch = await documentMatchesGroqFilter(
-            (targetRootValue || {}) as SanityDocument,
-            reference,
-            targetSchemaType.options,
+          const isMatch = await documentMatchesGroqFilter({
+            rootDocumentValue: (targetRootValue || {}) as SanityDocument,
+            referencedDocument: reference,
+            schemaTypeOptions: targetSchemaType.options,
             targetRootPath,
+            // todo: what to do here?
```

Comment:
I imagine we want to remove this? 😄 
Also, just to double check, do we actually have a "drafts" perspective? I remember there being something about raw but I just want to double check

Reply:
thanks for catching 😊 – you're right about not having a drafts-perspective. I'll fix.